File: task2/task2.cs
using System;
using System.IO;

class Program
{
    static void Main(string[] args)
    {
        //TASK2
        if (args.Length != 2)
        {
            Console.WriteLine("Enter circle and dot !");
            Environment.Exit(1);
        }
        float a, b; // Координаты центра круга
        float R; // Радиус круга

        using (StreamReader circle = new StreamReader(args[0]))
        {
            string[] circleCoords = circle.ReadLine().Split();
            a = float.Parse(circleCoords[0]);
            b = float.Parse(circleCoords[1]);
            R = float.Parse(circle.ReadLine());
        }

        using (StreamReader dot = new StreamReader(args[1]))
        {
            float x, y;
            while (!dot.EndOfStream)
            {
                string[] dotCoords = dot.ReadLine().Split();
                x = float.Parse(dotCoords[0]);
                y = float.Parse(dotCoords[1]);

                if ((x - a) * (x - a) + (y - b) * (y - b) == R * R) Console.WriteLine(0);
                if ((x - a) * (x - a) + (y - b) * (y - b) < R * R) Console.WriteLine(1);
                if ((x - a) * (x - a) + (y - b) * (y - b) > R * R) Console.WriteLine(2);
            }
        }
    }
}

File: task2/task2.py
import  sys

def main():
    if len(sys.argv) != 3:
        print("Enter circle and dot !")
        sys.exit(1)

    with open(sys.argv[1], 'r') as circle:
        a, b = map(float, circle.readline().split())
        R = float(circle.readline())

    with open(sys.argv[2], 'r') as dot:
        for line in dot:
            x, y = map(float, line.split())
            if (x - a) ** 2 + (y - b) ** 2 == R ** 2:
                print(0)
            elif (x - a) ** 2 + (y - b) ** 2 < R ** 2:
                print(1)
            elif (x - a) ** 2 + (y - b) ** 2 > R ** 2:
                print(2)


if __name__ == "__main__":
    main()